internal/response/merge.go

```diff
@@ -0,0 +1,73 @@
+package response
+
+func Merge(input []Response) []Response {
+	if input == nil {
+		return nil
+	}
+
+	if len(input) < 2 {
+		return input
+	}
```

Comment:
```suggestion
	if len(input) < 2 {
		return input
	}
```

don't need both cases (https://go.dev/play/p/dt-0n9rIl2K)